sources/platform/console/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Apify Console
+description: Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.
+sidebar_position: 1
+category: platform
+slug: /console
+---
+
+**Learn about Apify Console's easy account creation and user-friendly homepage for efficient web scraping management.**
+
+---
+
+## Sign Up
+
+To use Apify Console, you need to create Apify account. [Sign up here](https://console.apify.com/sign-up).
+
+![apify-console-sign-up](./images/console-signup.png)
+
+## Log In
+
+To log in to Apify Console, you can use one of the following methods:
+
+* Google Single Sign-On
+* GitHub Single Sign-On
+* The email and password set during account creation
+
+![apify-console-login](./images/console-login.png)
+
+## Homepage Overview
+
+![apify-console-homepage](./images/console-homepage.png)
+
+The Apify Console homepage provides an overview of our services. The header displays your account name and current plan level. The homepage features several sections:
+
+* **Get Started with Apify**: This section guides you to Apify Store. Here, you can explore existing Actors or create your own. A 'Show me how' button directs you to a guided tour on the Console.
+
+* **Recently Viewed**: This section displays Actors you have recently accessed. New users or those who haven't viewed any Actors will see suggestions instead.
+
+* **Suggested Actors for You**: Based on your and other users' recent activities, this section recommends Actors that might interest you.
+
+* **Actor Runs**: This section is divided into two tabs:
+  * **Recent**: View your latest Actor runs.
+  * **Scheduled**: Check your upcoming scheduled runs and tasks.
+
+Use the side menu to navigate other parts of the Apify Console easily.
+
+| Tab name | Description |
+|:---|:---|
+| [Store](/sources/platform/console/store.md)| Search for Actors that suit your web-scraping needs. |
+| [Actors](/sources/platform/actors/index.mdx)| View your recent runs, Actors developed by you, rented Actors, analytics & issues, as well as:<br/><br/> &bull; Saved tasks - where you can view your saved tasks or change any settings within them.<br/> &bull; Runs - where you can check on your previous runs. |
+| [Schedules](/sources/platform/schedules.md)| Schedule Actor runs & tasks to run at specified time. |
+| [Storage](/sources/platform/storage/index.md)| View stored results of your runs in various data formats. |
+| [Proxy](/sources/platform/proxy/index.md)| Setup proxy for your web-scrapers. |
```

Comment:
Suggestion: The page contains information about proxy usage, available proxies, and proxy credentials. I'd change the title to fit that.

Reply:
Yeah, sorry, I mean the description.

Reply:
Adjusted slightly to reflect ability to view proxy usage.

Reply:
I'd omit the "setup proxy for Actors" part, that's something that's usually done on Actor input level, not something you set globally.